src/test/sigencoding_tests.cpp

```diff
@@ -21,11 +21,17 @@ static valtype SignatureWithHashType(valtype vchSig, SigHashType sigHash) {
 static void CheckSignatureEncodingWithSigHashType(const valtype &vchSig,
                                                   uint32_t flags) {
     ScriptError err = ScriptError::OK;
-    BOOST_CHECK(CheckDataSignatureEncoding(vchSig, flags, &err));
 
     const bool hasForkId = (flags & SCRIPT_ENABLE_SIGHASH_FORKID) != 0;
+    // isTRKeySpend requires Schnorr sigs and BIP341 sighash
```

Comment:
Due to the piping for CDS, the `SCRIPT_ENABLE_SIGHASH_FORKID` enforces schnorr always as a side effect. This impacts CDS functionality (in tests) since the opcode script cannot be executed with this flag.